config/rzl.js

```diff
@@ -670,7 +680,99 @@ const config : Config = {
           delayedApply: true
         }
       ]
-    }
+    },
+    kitchen_counter_light: {
+      name: "Deckenlicht Theke",
+      position: [400, 440],
+      icon: "ceiling-light",
+      ui: [
+        {
+          type: "section",
+          text: "Lampe Eingang"
+        },
+        {
+          type: "slider",
+          min: 0,
+          max: 100,
+          text: "Helligkeit",
+          icon: "brightness-7",
+          topic: floalt.brightness("65544"),
+          delayedApply: true
+        },
+        {
+          type: "slider",
+          min: 0,
+          max: 100,
+          text: "Farbtemperatur",
+          icon: "weather-sunset-down",
+          topic: floalt.color("65544"),
+          delayedApply: true
+        },
+        {
+          type: "section",
+          text: "Lampe Hauptraum"
+        },
+        {
+          type: "slider",
+          min: 0,
+          max: 100,
+          text: "Helligkeit",
+          icon: "brightness-7",
+          topic: floalt.brightness("65543"),
+          delayedApply: true
+        },
+        {
+          type: "slider",
+          min: 0,
+          max: 100,
+          text: "Farbtemperatur",
+          icon: "weather-sunset-down",
+          topic: floalt.color("65543"),
+          delayedApply: true
+        }
+      ]
+    },
+    remotes: {
+      name: "Fernbedinungen",
+      position: [400, 348],
+      icon: "remote",
+      iconColor: (state) =>
+        ((state[tradfri_remote.low("65536")] == "true") || (state[tradfri_remote.low("65542")] == "true") || (state[tradfri_remote.low("65546")] == "true") || (state[tradfri_remote.low("65547")] == "true")) ? hex("#ff0000") : hex("#000000"),
```

Comment:
Please refactor! Drop the unneeded parenthesis. Also this line is way to long. Please break it into multiple lines.

You could also use the [_.every](https://lodash.com/docs/4.17.10#every) function of lodash to increase readability.